### PR TITLE
vsphere_virtual_machine: Add a test to catch empty UUID

### PR DESCRIPTION
**Pull request by gaeldb**
Prevent #558 (which is not a bug)

### CODE REVIEW

File: vsphere/data_source_vsphere_virtual_machine.go [hunk 1]
```diff
@@ -109,6 +109,10 @@ func dataSourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{
 		return fmt.Errorf("no configuration returned for virtual machine %q", vm.InventoryPath)
 	}
 
+	if props.Config.Uuid == "" {
+		return fmt.Errorf("configuration returned for virtual machine %q is scrappy: any UUID", vm.InventoryPath)
```
Comment (vancluever) [inline vsphere/data_source_vsphere_virtual_machine.go]: Can this error message be changed?  I'd suggest something like:

```
return fmt.Errorf("virtual machine %q does not have a UUID", vm.InventoryPath)
```

Comment (gaeldb) [reply vsphere/data_source_vsphere_virtual_machine.go]: Agree with you, let's go.

**Comment by gaeldb** [comment]
I changed error message
